requirements.txt

```diff
@@ -12,7 +13,8 @@ itsdangerous==0.24
 Jinja2>=2.10.1
 jsonschema==2.6.0
 MarkupSafe==1.0
-marshmallow==3.0.0rc7
+marshmallow==2.15.3
```

Comment:
@friendtocephalopods Just noticed this downgrade. Any reason?

Reply:
Ah, yes! Marshmallow-annotations is only compatible with 2.x.x; 3.0 hasn't released yet. This also makes it consistent with the other amundsen repos using marshmallow

Reply:
I see. @friendtocephalopods Have you seen this below? Seems related Marshmallow not being compatible..

`TypeError: attrs() got an unexpected keyword argument 'kw_only'" `

```
ts=2020-02-18T19:24:50.246Z uuid=4d7d5f2f-88d1-42e4-bf58-704883421eb1 app=amundsensearch name=gunicorn.error lvlname=ERROR lvlno=40 pid=25406 exc="Traceback (most recent call last):
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/arbiter.py\", line 574, in spawn_worker
    worker.init_process()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/workers/ggevent.py\", line 190, in init_process
    super(GeventWorker, self).init_process()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/workers/base.py\", line 127, in init_process
    self.load_wsgi()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/workers/base.py\", line 137, in load_wsgi
    self.wsgi = self.app.wsgi()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/app/base.py\", line 67, in wsgi
    self.callable = self.load()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/app/wsgiapp.py\", line 65, in load
    return self.load_wsgiapp()
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/app/wsgiapp.py\", line 52, in load_wsgiapp
    return util.import_app(self.app_uri)
  File \"/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/util.py\", line 357, in import_app
    __import__(module)
  File \"/srv/service/current/lyft_search_service/wsgi.py\", line 6, in <module>
    from search_service import create_app
  File \"/srv/venvs/service/trusty/service_venv_python3.6/lib/python3.6/site-packages/search_service/__init__.py\", line 15, in <module>
    from search_service.api.document import DocumentUserAPI, DocumentTableAPI, DocumentTablesAPI, DocumentUsersAPI
  File \"/srv/venvs/service/trusty/service_venv_python3.6/lib/python3.6/site-packages/search_service/api/document.py\", line 10, in <module>
    from search_service.models.table import TableSchema
  File \"/srv/venvs/service/trusty/service_venv_python3.6/lib/python3.6/site-packages/search_service/models/table.py\", line 9, in <module>
    @attr.s(auto_attribs=True, kw_only=True)
TypeError: attrs() got an unexpected keyword argument 'kw_only'" call_site=/srv/venvs/service/trusty/service_venv_python3.6/src/gunicorn/gunicorn/glogging.py:253 revision=ecd448d811 msg=Exception in worker process
```

Reply:
(diagnosed offline, no attrs in the requirements meant you got an old one. Adding to the amundsen common requirements in https://github.com/lyft/amundsencommon/pull/22 )